tasks/libs/pipeline_data.py

```diff
@@ -100,6 +100,15 @@ def get_failed_jobs(project_name, pipeline_id):
         re.compile(r'ERROR: The kitchen tests failed due to infrastructure failures\.'),
         FailedJobReason.KITCHEN,
     ),
+    # End to end tests EC2 Spot instances allocation failures
+    (
+        re.compile(r'Failed to allocate end to end test EC2 Spot instance after [0-9]+ attempts'),
+        FailedJobReason.EC2_SPOT,
+    ),
+    (
+        re.compile(r'Connection to [0-9]+\.[0-9]+\.[0-9]+\.[0-9]+ closed by remote host\.'),
```

Comment:
This error is quite generic, and might catch more logs than we'd expect. Let's keep an eye on these failures and check if they're getting miscategorized.

Reply:
I agree, I am not sure how to modify it though (not sure what is printing this in the first place). I can spend more time looking into it if you think this will cause trouble